lexos/static/js/scripts_upload.js

```diff
@@ -194,4 +194,51 @@ $(function () {
   if (window.File && window.FileList && window.FileReader) {
     Init()
   }
+
+  browserDetection()
 })
+
+function browserDetection () {
+  // Display a warning if using an unsupported browser
+
+  var browserWarningMsg = `You are currently using an unsupported browser.\
```

Comment:
Please use ```const``` if this variable does not get changed. Same for the couple declarations below.